.github/workflows/ci-tests.yml

```diff
@@ -91,10 +91,10 @@ jobs:
       - name: "conda environment cache"
         uses: ./.github/workflows/composite/conda-env-cache
         with:
-          cache_build: 0
+          cache_build: 1
           cache_period: ${{ env.CACHE_PERIOD }}
           env_name: ${{ env.ENV_NAME }}
-          install_packages: "cartopy nox pip"
+          install_packages: "cartopy tox'<4'"
```

Comment:
[Do we need Nox OR Tox anymore, now that we use lock files?!](https://github.com/catalyst-cooperative/pudl/pull/2968)

Reply:
> [Do we need Nox OR Tox anymore, now that we use lock files?!](https://github.com/catalyst-cooperative/pudl/pull/2968)

Having thought more, that was ill-informed, since Tox still offers neat inheritance tricks. But if we stick with tox-conda, we need to be comfortable sticking with the pin as well. Of course being pinned kinda undermines the idea of adopting something more popular.

Thanks @ESadek-MO and @HGWright for conversation on this topic.

Reply:
Also leaves us to potential security vulnerabilities:

https://github.com/tox-dev/tox/issues/2524#issuecomment-1298934474